MAINTAINERS.md

```diff
@@ -14,16 +14,9 @@
 |------|--------|------|
 | Tech Lead | Jamie Monserrate | [jamiemonserrate](https://github.com/jamiemonserrate) |
 | Tech Lead  | Shailesh Pant | [pshail](https://github.com/pshail) |
-| Eng Management | Gavin Lu |[luguang](https://github.com/luguang) |
 | Eng Management | Sudheesh Sudhakaran |[sudheesh-csp](https://github.com/sudheesh-csp) |
 | Eng Management | Vijayakumar Arumuga Nadar |[Vijayakumar0328](https://github.com/Vijayakumar0328) |
-| Product Management | Disha Chopra |[dishac](https://github.com/dishac) |
-| Engineering | Aleksandar Georgiev |[georgievaVMW](https://github.com/georgievaVMW) |
 | Engineering | Anusha Hegde |[anusha94](https://github.com/anusha94) |
 | Engineering | Dharmjit Singh |[dharmjit](https://github.com/dharmjit/) |
-| Engineering | Hui Chen |[huchen2021](https://github.com/huchen2021) |
```

Comment:
I am not sure of having an emeritus contributors list. But I think we can have one. BYOH wouldn't be what it is without the initial contributions of Hui, Yixing, Wuming, Rousko, and others :) 
I'd leave it to you though.

Reply:
I thought about this and it definitely makes sense to keep all the core contributors to date. I will take some hints from other OSS projects and any guidance I can get. Keeping a emeritus contributors list also seems like an option.